lib/facter/confluence_version.rb

```diff
@@ -5,7 +5,7 @@
     if confluence_process.nil?
       'unknown'
     else
-      confluence_process.scan(%r{\d+\.\d+\.\d+}).first
+      confluence_process.scan(%r{atlassian-confluence-(\d+\.\d+\.\d+)}).first.first
```

Comment:
Want verification on this. What confluence versions have you tested this fact with?